src/olympia/editors/templates/editors/reviewlog.html

```diff
@@ -46,13 +46,15 @@
                   {% else %}
                     {% set channel = 0 %}
                   {% endif %}
-                  {% if channel == amo.RELEASE_CHANNEL_LISTED %}
+                  {% if item.action == amo.LOG.REJECT_CONTENT.id or item.action == amo.LOG.APPROVE_CONTENT.id %}
```

Comment:
I guess this works until we start content reviewing unlisted add-ons, heh.

Reply:
We won't. Unlisted add-ons by design do not apply to content policies.